hera_mc/tests/test_geo_location.py

```diff
@@ -143,7 +143,8 @@ def test_find_antenna_station_pair(self):
         self.assertTrue(ant == 'A23')
         ant, rev = self.h.find_antenna_at_station('BB23', 'now')
         self.assertEqual(ant, None)
-        self.assertRaises(ValueError, self.h.find_antenna_at_station, 'HH68', 'now')
```

Comment:
was this testing the error that's become a warning? We should check that the warning is raised using the `checkWarnings` method in `test/__init__.py`

Reply:
![Untitled-1](https://user-images.githubusercontent.com/4715126/58502803-2927b600-813c-11e9-8b3d-4f3e9f408a43.png)

Reply:
See above for sample data.  The purple dashed line is from cm_dataview: "ants_by_day" and the black and green solid lines are form cm_dataview: "connected_by_day".  It is a brute force slow step through time.

Reply:
I've added the checkWarning test

Reply:
good. This plot will never be seen because it's inside a code review thread. Can you put it directly on the PR as a new comment?